### DISCUSSION TITLE
Clipping distance

### DISCUSSION THREAD

**Discussion by JD31** [pelicanmapping/osgearth#1904] [Support Q&A] [answered]
Hi Community!

I am in a particular case where I need to desactive the depth test.
It works as I expect, and - as expected - I can see feature models which are at the opposite side of the globe (not facing the camera).
Activating back face culling culls only parts of the features but not all.
I guess the reason is that I speak of a very big LineDrawable model, where triangles are expanded in screen space in the shader...

My question is simply : Is it possible to define a clip distance where all bounding spheres over this distance would be automatically culled ?

Thanks,
JD

**Comment by gwaldron** [comment, marked as answer]
JD,
Normally the camera is set up to automatically compute a far clipping plane based on the visible geometry. But you can change that and force it to a value.

For your `osg::Camera` you can say something like
```c++
camera->setComputeNearFarMode(osg::CullSettings::DO_NOT_COMPUTE_NEAR_FAR);
camera->setProjectionMatrixAsPerspective(vfov, aspect_ratio, near_clip, far_clip);
```
Hope this helps.
